### PR TITLE
TASK: Add composer scripts to run tests and psalm

### PR DESCRIPTION
**Pull request by albe**
Run Unit-Tests: `composer test-unit`
Run Functional Tests: `composer test-functional`
Run static analysis: `composer test-static`
Update psalm baseline: `composer psalm-baseline-update`

### CODE REVIEW

**Comment by kitsunet** [comment]
mmm, I would definitely prefer to prefix both (to leave a combined "run all the tests" open. So IMHO

`test-unit`
`test-functional`
`test-static` (psalm, because no tool name)
`test-static-baseline-update`

**Comment by albe** [comment]
> mmm, I would definitely prefer to prefix both (to leave a combined "run all the tests" open. So IMHO
> 
> `test-unit`
> `test-functional`
> `test-static` (psalm, because no tool name)
> `test-static-baseline-update`

Good point. The `baseline update` thing is very psalm specific though (and mostly needed by maintainers to "fix" the psalm errors that cannot be fixed in code), so maybe call that one with `psalm-baseline-update`?

**Comment by kitsunet** [comment]
Yes, makes sense (the psalm-baseline-update)

File: composer.json [hunk 1]
```diff
@@ -30,7 +30,11 @@
         "post-update-cmd": "Neos\\Flow\\Composer\\InstallerScripts::postUpdateAndInstall",
         "post-install-cmd": "Neos\\Flow\\Composer\\InstallerScripts::postUpdateAndInstall",
         "post-package-update": "Neos\\Flow\\Composer\\InstallerScripts::postPackageUpdateAndInstall",
-        "post-package-install": "Neos\\Flow\\Composer\\InstallerScripts::postPackageUpdateAndInstall"
+        "post-package-install": "Neos\\Flow\\Composer\\InstallerScripts::postPackageUpdateAndInstall",
+        "test": "phpunit -c Build/BuildEssentials/PhpUnit/UnitTests.xml",
+        "test-func": "phpunit -c Build/BuildEssentials/PhpUnit/FunctionalTests.xml",
+        "psalm": "psalm --config=Packages/Framework/psalm.xml --show-info=false",
+        "psalm-baseline": "psalm --config=Packages/Framework/psalm.xml --set-baseline=Packages/Framework/psalm-baseline.xml"
```
Comment (albe) [inline composer.json:37]:
```suggestion
        "test-unit": "phpunit -c Build/BuildEssentials/PhpUnit/UnitTests.xml",
        "test-functional": "phpunit -c Build/BuildEssentials/PhpUnit/FunctionalTests.xml",
        "test-static": "psalm --config=Packages/Framework/psalm.xml --show-info=false",
        "psalm-baseline-update": "psalm --config=Packages/Framework/psalm.xml --set-baseline=Packages/Framework/psalm-baseline.xml"
```

**Pull request merged**